src/Compilers/CSharp/Test/Semantic/Semantics/NullCoalescingAssignmentTests.cs

```diff
@@ -0,0 +1,1408 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.UnitTests;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.Semantic.UnitTests.Semantics
+{
+    [CompilerTrait(CompilerFeature.NullCoalescingAssignment)]
+    public class NullCoalescingAssignmentTests : CompilingTestBase
+    {
+        [Fact]
+        public void LocalLvalue()
+        {
+
+            CompileAndVerify(@"
+using System;
+public class C
+{
+    public static void Main()
+    {
+        int? i1 = null;
```

Comment:
>null [](start = 18, length = 4)

If you don't have that already, can you also test with a non-null value? Then `GetInt` should not be invoked.